moduleroot/.github/settings.yml.erb

```
@@ -0,0 +1,52 @@
+<% if @configs['repository'] -%>
+repository:
+<%   @configs['repository'].each do |key, value| -%>
+  <%= key %>: <%= value %>
+<%   end -%>
+<% end -%>
+<% if @configs['labels'] -%>
+
+labels:
+<%   @configs['labels'].each do |label| -%>
+  - name: <%= label['name'] %>
+<%     label.keys.each do |k| -%>
+<%       next if k == 'name' -%>
+    <%= k %>: <%= label[k] %>
+<%     end -%>
+<%   end -%>
+<% end -%>
+<% if @configs['branches'] -%>
+
+branches:
+<%   @configs['branches'].each do |branch| -%>
+  - name: <%= branch['name'] %>
+<%     branch.keys.each do |k| -%>
```

You can also iterate over the tuples
```suggestion
<%     branch.keys.each do |key, value| -%>
```

This avoids the whole `branch[k]` part.